src/index.html

```diff
@@ -30,6 +30,7 @@
 
         <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
         <script type="module" src="./js/index.tsx"></script>
+        <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
This shouldn't be necessary.

The styles and logic should be bundled in the app based off your Tailwind installation and config.

Did you follow the Vite guide? https://tailwindcss.com/docs/guides/vite

Reply:
Yeah, I followed the guide but the styling wouldn't apply. It turns out the create sample file selector styling was because of the styled version of `CompactScrollList`. I tried some other methods too but this one was the only one that seemed to apply the styling changes